services/common/logger/zap.go

```diff
@@ -265,3 +268,22 @@ func (log *MerakLog) Flush() error {
 	}
 	return e
 }
+
+func LevelEnvParser(level string) Level {
+	switch strings.ToLower(level) {
+	case "debug":
+		return DEBUG
+	case "info":
+		return INFO
+	case "warn":
+		return WARN
+	case "error":
```

Comment:
@phudtran does this change affect the usage of logger wrapper? I knew @zzengcs already use logger wrapper in her code, like: `Logger, err = logger.NewLogger(logger.DEBUG, "Merak-topo")`

Reply:
No this will not affect her change.
This is an additional feature if you want to initialize a new logger with a log level defined by a string from an env variable.